src/components/Card/Card.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { ReactNode } from "react";
+import styled from 'styled-components';
+import { BOX_BORDER_RADIUS } from '../../globalStyles/sharedStyles/box';
+
+const CardComponent = styled.div`
+  width: 100%;
+  padding: 36px 48px;
```

Comment:
from task description - _padding - as on the Vazco website components_
padding from Vazco website - `padding: 27px 22px 24px;`

Reply:
We discussed with @caluskrzysztof that the padding should be made a bit bigger in the end